src/commands/readme.ts

```diff
@@ -131,7 +131,7 @@ USAGE
   commands(config: Config.IConfig, commands: Config.Command[]): string {
     return [
       ...commands.map(c => {
-        let usage = this.commandUsage(c)
+        let usage = this.commandUsage(config, c)
```

Comment:
@amphro is being modified anywhere else?  If not, I would change them to `const` (same for the `title` variable)